test/integration/targets/jboss/defaults/main.yml

```
@@ -0,0 +1,4 @@
+jboss_root: /opt/wildfly
```

Use a temporary directory managed by the test infrastructure if possible:

```suggestion
jboss_root: '{{ remote_tmp_dir }}/opt/wildfly'
```